---
layout: page
slide_id: 3
---
<div class="card-block bg-faded">
<h1 id="workshop-program">Workshop Program</h1>
<p>The GSC23 Bangkok meeting will be followed by two concurrent industry led workshops.
Registration for the workshops are independent of the conference registration, but is primarily aimed at conference attendees. Each workshop will last a full day, and be co-located with the main conference in the SiMR building. </p><br>
<p><strong>MGnify workshop venue:</strong> 1st floor, SiMR building (same room as the conference room)<br>
<strong>Nanopore workshop venue:</strong> room 304-305, 3rd floor, SiMR building</p><br>
<p>The registration costs for each workshop will be 200USD per person to cover the tutors expenses, venue hire, food and sundries. As they are being run at the same time, participants may only register for one of the workshops. </p>
<p>Each workshop is expected to run from 9am to 5pm with morning and afternoon coffee breaks, and a lunch break where food will be provided. Due to the hands-on nature of these training workshops the number of participants will be restricted to a maximum of 30 people to ensure that everyone receives adequate tutelage. Please ensure you meet the relevant prerequisites for the course before signing up. </p>
<br><h1> Nanopore Workshop Program</h1>
<img src="assets/images/nanopore-logo.png" alt="">
<p><b>Location:</b> Room 304-305, SiMR Building, Siriraj Hospital Mahidol University, 2 Thanon Wang Lang, Siriraj, Bangkok Noi, Bangkok 10700, Thailand</p>
<p><b>Trainers:</b> Scott Tighe,Tip Wongsurawant and Nanopore Staff.</p>
<p><a href="https://nanoporetech.com/ target=_blank">Oxford Nanopore Technologies Limited</a> is a UK-based company which develops and sells nanopore sequencing products (including the portable DNA sequencer, MinION) for the direct, electronic analysis of single molecules.</p>
<p>This hands-on workshop will provide attendees with a critical understanding of Oxford Nanopore sequencing technology through a comprehensive laboratory and analysis workflow approach. The workshop will provide a background understanding of the sequencing technology, QC, and sequencer set up with laboratory training including “hands-on“ individually generated sequence ready barcoded libraries, loading flow cells, and operating the MinKNOW sequencing software. Basic data analysis with EPI2ME software will allow participants to analyze and view the results. The program will consist of a morning session where participants will prepare actual sequencing libraries, load the MK1C MinION flow cell and initiate sequencing. An afternoon session will include understanding the QC outputs and Introduction to basecalling and Epi2ME analysis workflows.</p><br>
<p><b>Prerequisites</b> include basic laboratory safety procedures, low volume pipette operations, and basic understanding of DNA sequencing.</p>
<br>
<p><strong>Topics covered:</strong></p>
<ul>
<li>Introduction and Overview of Technology</li>
<li>Library Synthesis and Quality Control</li>
<li>Loading the Nanopore Flow Cell</li>
<li>Operating MinKNOW Software</li>
<li>Sequence Operations of MK1C</li>
<li>Sequence Monitoring Metrics</li>
<li>Post Sequencing Data Analysis</li>
</ul>
<br>
<h1> MGnify Workshop Program</h1>
<img src="assets/images/MGnify-logo.png" width="250" height="125">
<p><b>Location:</b> 1st Floor, SiMR Building, Siriraj Hospital Mahidol University, 2 Thanon Wang Lang, Siriraj, Bangkok Noi, Bangkok 10700, Thailand</p>
<p><b>Trainers:</b> Rob Finn, Lorna Richardson, Tanya Gurbich (EMBL-EBI,UK)</p>
<br>
<p><a href="https://www.ebi.ac.uk/metagenomics" target=_blank>MGnify</a> is a freely available resource, provided by EMBL-EBI, for the analysis and archiving of microbiome-derived sequencing data. Users can request assembly of raw-read data, and taxonomic and functional analysis of publicly available (or pre-publication) datasets available in ENA. MGnify also provides microbiome-derived genome catalogues for multiple biomes, and a non-redundant protein database of over 3 billion sequences.</p>
<p>Two half-day workshops (a mixture of hands-on practicals and lectures) are available on Friday 11th August, provided by the MGnify resource. In the morning there will be an introductory session for users new to the field and/or the resource, and in the afternoon a more advanced session. More details are provided below about what will be covered in each session. Registration will be for both sessions, but delegates should specify which sessions they will attend - either one or both of the sessions, as they prefer based on their experience.</p><br>
<p>Please note that attendees will be expected to bring along their own laptop computer for use within the practical sessions. All practical aspects will be browser-based and so the only software required for the course will be access to a recent version of either Chrome, Firefox, or Safari.</p><br>
<p><strong>Introductory session (9:00-12:00)</strong></p>
<p><strong>Topics covered:</strong></p>
<ul>
<li>What is MGnify?</li>
<li>Analysis and results of metabarcoding and metagenomic/metatranscriptic datasets</li>
<li>Assembly of metagenomic data - the advantages and disadvantages</li>
<li>Submitting data to ENA for analysis by MGnify</li>
<li>MGnify quality control and data cleaning steps</li>
<li>Why metadata is important</li>
<li>Searching MGnify and accessing results</li>
</ul>
<p><strong>Advanced session (14:00-17:00)</strong></p>
<p><strong>Topics covered:</strong></p>
<ul>
<li>Generating MAGs from metagenomic data</li>
<li>Biome-specific genome catalogues</li>
<li>Comparing new MAGs against genome catalogues</li>
<li>Programmatic access to MGnify</li>
<li>Downstream analysis in Jupyter notebooks, e.g. comparing across datasets</li>
</ul>
<br>
</div>
